### PR TITLE
Fix drag and drop example

### PR DESCRIPTION
**Pull request by MatthieuDartiailh**
Depends on #18

### CODE REVIEW

File: examples/widgets/drag_and_drop.enaml [hunk 1]
```diff
@@ -97,13 +98,14 @@ enamldef Main(Window):
         pass
     Container:
         constraints = [
-            hbox(vbox(lbl1, lbl2, spacer), target),
+            hbox(vbox(lbl1, lbl2), target),
         ]
         DragLabel: lbl1:
             text = 'Drag Me 1'
-            data = 'small\ndata'
+            data = b'small\ndata'
```
Comment (tillbey) [inline examples/widgets/drag_and_drop.enaml:105]: In this case maybe better
```python
bytes('small\ndata')
```
Else `\n` stays literal.

File: examples/widgets/drag_and_drop.enaml [hunk 2]
```diff
@@ -97,13 +98,14 @@ enamldef Main(Window):
...
         DragLabel: lbl2:
             text = 'Drag Me 2'
-            data = '\n'.join([str(i) for i in range(100)])
+            data = b'\n'.join([bytes(i) for i in range(100)])
```
Comment (tillbey) [inline examples/widgets/drag_and_drop.enaml]: same here

File: examples/widgets/drag_and_drop.enaml (hunk 1 above)
Comment (MatthieuDartiailh) [reply examples/widgets/drag_and_drop.enaml:105]: Not sure what you mean by stay literal... if it is that it is translated in two characters then I think the code for b-string may be wrong.

Comment (MatthieuDartiailh) [reply examples/widgets/drag_and_drop.enaml:105]: With the latest fix from #18 (this branch has been rebased on it) I think everything now works as expected.

Comment (tillbey) [reply examples/widgets/drag_and_drop.enaml:105]: Yes, I tested it. This works

**Pull request merged**